### PR TITLE
Updates to Azure installation

### PR DESCRIPTION
**Pull request by mhausenblas**
As of today, DC/OS 1.8.4 is available in the Azure [Marketplace](https://azure.microsoft.com/en-us/marketplace/partners/mesosphere/dcosdcos/). The policy is: the [current docs](https://dcos.io/docs/latest/) always contain the detailed setup instructions for the current version (ATM 1.8.4.) and the old/upcoming docs (ATM 1.7 and 1.9) point to the ARM templates via [downloads.dcos.io](https://downloads.dcos.io/dcos/).

Changes in this PR according to above policy:

- updates the outdates 1.8 install instructions (incl. new OAuth enable feature)
- updates 1.7 and 1.9 docs, replacing step-by-step instructions with ARM templates references

@pyronicide can you pls review and publish, I'd like to blog about it and spread the word ASAP since many folks are waiting for this.

### CODE REVIEW

File: 1.7/administration/installing/cloud/azure.md [hunk 1]
```diff
@@ -4,7 +4,7 @@ nav_title: Azure
 menu_order: 1
 ---
 
-This document explains how to install DC/OS by using the Azure Marketplace.
+This document explains how to install DC/OS 1.7 through the Azure Marketplace.
```
Comment (pyronicide) [inline 1.7/administration/installing/cloud/azure.md]: Using the Azure Resource Manager templates?

File: 1.7/administration/installing/cloud/azure.md [hunk 2]
```diff
@@ -24,95 +24,7 @@ Also, to access nodes in the DC/OS cluster you will need `ssh` installed and con
...
-
-![Deploying DC/OS template](../img/dcos-azure-marketplace-step1e.png)
-
-After the deployment succeeded, click on the resource group (`mydcoscluster` here) and you should get to the resource group. If you don't see it, try searching for your research group and if the deployment failed, delete the deployment and the resource group and start again:
-
-![DC/OS template successfully deployed](../img/dcos-azure-marketplace-step1f.png)
-
-Congratulations, you have now deployed DC/OS by using an Azure Resource Manager template! Next we will access the cluster.
-
-## Step 2: Accessing DC/OS
```
Comment (pyronicide) [inline 1.7/administration/installing/cloud/azure.md]: We should probably keep this whole step2.

File: 1.7/administration/installing/cloud/azure.md [hunk 3]
```diff
@@ -24,95 +24,7 @@ Also, to access nodes in the DC/OS cluster you will need `ssh` installed and con
...
-
-```bash
-$ ssh azureuser@dcosmaster.westus.cloudapp.azure.com -p 2200 -L 8000:localhost:80
-```
-
-Now you can visit `http://localhost:8000` on your local machine and view the DC/OS Dashboard.
-
-![DC/OS dashboard](../img/ui-dashboard.gif)
-
-### Caveats
```
Comment (pyronicide) [inline 1.7/administration/installing/cloud/azure.md]: This seems relevant.

File: 1.7/administration/installing/cloud/azure.md [hunk 4]
```diff
@@ -24,95 +24,7 @@ Also, to access nodes in the DC/OS cluster you will need `ssh` installed and con
...
-$ mkdir -p dcos && cd dcos &&
-$ curl -O https://downloads.dcos.io/dcos-cli/install-optout.sh && \
-   bash ./install-optout.sh . http://localhost && \
-   source ./bin/env-setup
-
-# Now you can use the DC/OS CLI:
-$ dcos package search
-```
-
-## Tear Down the DC/OS cluster
```
Comment (pyronicide) [inline 1.7/administration/installing/cloud/azure.md]: Feels relevant again.

File: 1.8/administration/installing/cloud/azure.md [hunk 5]
```diff
@@ -4,19 +4,19 @@ nav_title: Azure
 menu_order: 1
 ---
 
-This document explains how to install DC/OS by using the Azure Marketplace.
+This document explains how to install DC/OS 1.8 through the Azure Marketplace.
 
-You can alternatively skip the marketplace and install directly using the <a href="https://downloads.dcos.io/dcos/stable/azure.html" target="_blank">DC/OS templates for Azure Resource Manager</a>.
+You can alternatively skip the Azure Marketplace and install directly using the [Azure Resource Manager templates](https://downloads.dcos.io/dcos/stable/commit/e64024af95b62c632c90b9063ed06296fcf38ea5/azure.html) provided.
```
Comment (pyronicide) [inline 1.8/administration/installing/cloud/azure.md]: Can we leave this link `stable` for now? I'd like to not have to update this on a regular basis.

File: 1.8/administration/installing/cloud/azure.md [hunk 6]
```diff
@@ -4,19 +4,19 @@ nav_title: Azure
...
 
 # System requirements
 
 ## Hardware
 
-To [use](/docs/1.8/usage/) all of the services offered in DC/OS, you should choose at least five Mesos Agents using `Standard_D2` [Virtual Machines](https://azure.microsoft.com/en-us/pricing/details/virtual-machines/), which is the default size in the DC/OS Azure Marketplace offering.
+To [use](/docs/1.8/usage/) all of the services offered in DC/OS, you should choose at least five Mesos Agents using `Standard_D2` [Virtual Machines](https://azure.microsoft.com/en-us/pricing/details/virtual-machines/).
 
-Selecting smaller-sized VMs is not recommended, and selecting fewer VMs will likely cause certain resource-intensive services such as distributed datastores not to work properly (from installation issues to operational limitations).
+Selecting smaller-sized VMs is not recommended and selecting fewer VMs will likely cause certain resource-intensive services such as distributed datastores not to work properly, ranging from installation issues to operational limitations.
```
Comment (pyronicide) [inline 1.8/administration/installing/cloud/azure.md]: Note, `Standard_D2` is the smallest size that you can use.

File: 1.9/administration/installing/cloud/azure.md [hunk 7]
```diff
@@ -4,17 +4,15 @@ nav_title: Azure
 menu_order: 1
 ---
 
-This document explains how to install DC/OS by using the Azure Marketplace.
-
-You can alternatively skip the marketplace and install directly using the <a href="https://downloads.dcos.io/dcos/stable/azure.html" target="_blank">DC/OS templates for Azure Resource Manager</a>.
+This document explains how to install DC/OS 1.9 through the Azure Marketplace.
```
Comment (pyronicide) [inline 1.9/administration/installing/cloud/azure.md]: Let's remove this doc from 1.9 for now (there aren't any templates for 1.9 yet).

**Comment by joel-hamill** [comment]
@mhausenblas https://dcosjira.atlassian.net/browse/DCOS-596

**Pull request merged**